paywall/src/data-iframe/cache/localStorage.js

```diff
@@ -97,15 +80,12 @@ export async function put({
   accountAddress = nullAccount,
 }) {
   ensureLocalStorageAvailable(window, `save ${type} in cache`)
-  const key = storageId(networkId, accountAddress)
-  const container = getContainer(window, key, type)
+  const key = storageId(networkId, accountAddress, type)
   if (value === undefined) {
```

Comment:
this is more natural behavior: if you `put` something, it saves it. If you put `undefined`, we clear the localStorage entry.